app/controllers/report_controller/reports.rb

```diff
@@ -373,6 +373,7 @@ def check_tabs
         elsif @edit[:new][:model] == "Chargeback"
           unless @edit[:new][:cb_show_typ] &&
                  ((@edit[:new][:cb_show_typ] == "owner" && @edit[:new][:cb_owner_id]) ||
+                  (@edit[:new][:cb_show_typ] == "tenant" && @edit[:new][:cb_tenant_id]) ||
```

Comment:
@aljesusg  can you move this huge condition (and if it is possible change it somehow for better readibility) to a method valid_chargeback_fields
then we can get here
```
elsif @edit[:new][:model] == "Chargeback" && !valid_chargeback_fields
     add_flash..
     ....
```

what do you think ?